andela/selene-ah-backend
# #162342178 Write tests for the User Model

#### What does this PR do?
Test the User Model
#### Description of Task to be completed?
- test the existence of the user model
- test the existence of the user model properties
- check for the uniqueness of the email and username properties
- test the associations for the user model
- refactor the user model association definitions
- delete the index file on the test directory
#### How should this be manually tested?
- clone the repository e.g on your command-line and  type:  git clone https://github.com/andela/selene-ah-backend.git
- type npm install
- type npm test
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
[#162342178](https://www.pivotaltracker.com/story/show/162342178)
#### Screenshots (if appropriate)
none
#### Questions:
none
